src/lib/openapi/spec/advanced-playground-request-schema.ts

```diff
@@ -0,0 +1,48 @@
+import { FromSchema } from 'json-schema-to-ts';
+import { ALL } from '../../types/models/api-token';
+import { sdkContextSchema } from './sdk-context-schema';
+
+export const advancedPlaygroundRequestSchema = {
+    $id: '#/components/schemas/advancedPlaygroundRequestSchema',
+    description:
+        'Data for the playground API to evaluate toggles in advanced mode with environment and context multi selection',
+    type: 'object',
+    required: ['environments', 'context'],
+    properties: {
+        environments: {
```

Comment:
this one differs from the old playground because we support and array of envs, not just one